iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/ContentIntegrationTests.cpp

```diff
@@ -5465,6 +5465,70 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, ContentDescriptorIsRemovedF
     EXPECT_NE(descriptor1, descriptor2);
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsitest
++---------------+---------------+---------------+---------------+---------------+------*/
+DEFINE_SCHEMA(ContentDescriptorIsCachedWhenNotRelatedRulesetVariableChanges, R"*(
+    <ECEntityClass typeName="A">
+        <ECProperty propertyName="Property" typeName="string" />
+    </ECEntityClass>
+)*");
+TEST_F(RulesDrivenECPresentationManagerContentTests, ContentDescriptorIsCachedWhenNotRelatedRulesetVariableChanges)

```

Comment:
I'd rename the tests to:
- `FindsCachedDescriptorWhenAllRelatedRulesetVariablesMatch`
- `CreatesNewDescriptorWhenRelatedRulesetVariablesDontMatch`